library/tinkerpop/src/test/resources/gafferpop-tinkerpop-modern.properties

```diff
@@ -15,6 +15,7 @@
 #
 gremlin.graph=uk.gov.gchq.gaffer.tinkerpop.GafferPopGraph
 gaffer.graphId=graph1
-gaffer.storeproperties=conf/gafferpop/gaffer/store.properties
-gaffer.schemas=conf/gafferpop/gaffer/schema/
+gaffer.storeproperties=${project.basedir}/src/test/resources/gaffer/store.properties
+gaffer.schemas=${project.basedir}/src/test/resources/gaffer/schema/
```

Comment:
Why this change?

Reply:
This change goes with the change to `pom.xml`. It replaces the path with a full path. Though I don't see why this is required.